app/build.gradle

```diff
@@ -61,15 +75,17 @@ android {
             buildConfigField("boolean", "ACRA_ENABLED", "false")
             buildConfigField("String", "ACRA_BASIC_AUTH_LOGIN", "null")
             buildConfigField("String", "ACRA_BASIC_AUTH_PASSWORD", "null")
+            signingConfig signingConfigs.debug
         }
         release {
-            buildConfigField("boolean", "ACRA_ENABLED", "true")
-            def config = tryReadConfig("release.env")
-            buildConfigField("String", "ACRA_BASIC_AUTH_LOGIN", generatedString(config.login))
-            buildConfigField("String", "ACRA_BASIC_AUTH_PASSWORD", generatedString(config.password))
+//            def config = tryReadConfig("release.env")
+            buildConfigField("boolean", "ACRA_ENABLED", "false")
+            buildConfigField("String", "ACRA_BASIC_AUTH_LOGIN", "null")
+            buildConfigField("String", "ACRA_BASIC_AUTH_PASSWORD", "null")
 
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.debug
```

Comment:
I think all this should be undone pre-merge (it'll build on your machine even without `release.env`, so it's safe to just leave uncommented all the time)

Reply:
def config = tryReadConfig("release.env") throws this warning:
https://i.imgur.com/WOHZDea.png

Reply:
You also need to revert the lines which were reading values from that (and probably the signing config lines as well)